backend/src/main/java/reviewme/review/service/mapper/ReviewListMapper.java

```diff
@@ -16,23 +17,27 @@
 @RequiredArgsConstructor
 public class ReviewListMapper {
 
-    private final OptionItemRepository optionItemRepository;
     private final ReviewRepository reviewRepository;
+    private final OptionItemRepository optionItemRepository;
 
     private final ReviewPreviewGenerator reviewPreviewGenerator = new ReviewPreviewGenerator();
 
     public List<ReviewListElementResponse> mapToReviewList(ReviewGroup reviewGroup) {
-        return reviewRepository.findReceivedReviewsByGroupId(reviewGroup.getId())
+        List<OptionItem> categoryOptionIds = optionItemRepository.findAllByOptionType(OptionType.CATEGORY);
+        return reviewRepository.findAllByGroupId(reviewGroup.getId())
                 .stream()
-                .map(this::mapToReviewListElementResponse)
+                .map(review -> mapToReviewListElementResponse(review, categoryOptionIds))
                 .toList();
     }
 
-    private ReviewListElementResponse mapToReviewListElementResponse(Review review) {
-        List<OptionItem> categoryOptionItems = optionItemRepository
-                .findByReviewIdAndOptionType(review.getId(), OptionType.CATEGORY);
+    private ReviewListElementResponse mapToReviewListElementResponse(Review review,
+                                                                     List<OptionItem> categoryOptionItems) {
+        Set<Long> checkBoxOptionIds = review.getAllCheckBoxOptionIds();
+        List<OptionItem> categoryOptionItemsByReview = categoryOptionItems.stream()
+                .filter(optionItem -> checkBoxOptionIds.contains(optionItem.getId()))
+                .toList();
```

Comment:
categoryOptionItemsByReview를 만들고 그걸로 response를 만들지않고,
한번의 스트림으로 바로 response까지 만들 수 있겠어요! 깔꼼깔꼼!!

```suggestion
        List<ReviewCategoryResponse> categoryResponses = categoryOptionItems.stream()
                .filter(optionItem -> checkBoxOptionIds.contains(optionItem.getId()))
                .map(optionItem -> new ReviewCategoryResponse(optionItem.getId(), optionItem.getContent()))
                .toList();
```